uvm/js/common/ungrid/GridController.js

```diff
@@ -525,6 +534,18 @@ Ext.define('Ung.cmp.GridController', {
                 if (fieldConfig !== undefined && (fieldConfig.validator || fieldConfig.vtype || !fieldConfig.allowBlank)) {
                     var validationErrorMsg = null;
 
+                    if(fieldConfig.hasOwnProperty("minValue")){
+                        if(fieldValue < fieldConfig.minValue){
+                            validationErrorMsg = 'The minimum value for this field is ' + fieldConfig.minValue + "."; 
```

Comment:
Because field field positions can be in different places for non-English languages, we should use the the following format:
Ext.String.format('This {0} Server is entitled to Live Support.'.t(), paramtername)

Reply:
changed